src/main/resources/twilio-go/api.mustache

```diff
@@ -39,32 +39,39 @@ type {{{nickname}}}Params struct {
 {{/hasOptionalParams}}
 
 // {{operationId}}{{#summary}} {{{.}}}{{/summary}}{{^summary}} Method for {{operationId}}{{/summary}}
-//
 {{#notes}}
-// {{notes}}
 //
+// * {{notes}}
 {{/notes}}
 {{#allParams}}
+{{^vendorExtensions.x-is-account-sid}}
 {{#required}}
-// param: {{paramName}}{{#description}} {{{.}}}{{/description}}
 //
+// * @param: {{paramName}}{{#description}} {{{.}}}{{/description}}
 {{/required}}
+{{/vendorExtensions.x-is-account-sid}}
 {{/allParams}}
 {{#hasOptionalParams}}
-// param: optional nil or *{{{nickname}}}Params - Optional Parameters:
 //
+// * @param: optional nil or *{{{nickname}}}Params - Optional Parameters:
 {{#optionalParams}}
-// param: "{{paramName}}" ({{#isNullable}}*{{/isNullable}}{{{dataType}}}) - {{#description}}{{{.}}}{{/description}}
 //
+// * @param: "{{paramName}}" ({{#isNullable}}*{{/isNullable}}{{{dataType}}}) - {{#description}}{{{.}}}{{/description}}
 {{/optionalParams}}
 {{/hasOptionalParams}}
 {{#returnType}}
-// return: {{{returnType}}}
+//
+// * @return: {{{returnType}}}
 {{/returnType}}
-func (c *{{{classname}}}Service) {{{nickname}}}({{#allParams}}{{#required}}{{paramName}} {{{dataType}}}{{^last}}, {{/last}}{{/required}}{{/allParams}}{{#hasOptionalParams}}params *{{{nickname}}}Params{{/hasOptionalParams}}) ({{#returnType}}*{{{returnType}}}, {{/returnType}}error) {
+func (c *{{{classname}}}Service) {{{nickname}}}({{#allParams}}{{#required}}{{^vendorExtensions.x-is-account-sid}}{{paramName}} {{{dataType}}}{{^last}}, {{/last}}{{/vendorExtensions.x-is-account-sid}}{{/required}}{{/allParams}}{{#hasOptionalParams}}params *{{{nickname}}}Params{{/hasOptionalParams}}) ({{#returnType}}*{{{returnType}}}, {{/returnType}}error) {
```

Comment:
We need to also allow the accountSid to be passed in as part of the `params` since it's optional.

Reply:
That should still be allowed though. This change says if the parameter is [required (a path parameter)](https://github.com/twilio/twilio-oai-generator/blob/subaccount-go/src/main/java/com/twilio/oai/TwilioGoGenerator.java#L23) and if it is equal to 'AccountSid', then do not include it as a func parameter but I am not touching the optional parameters.

Reply:
I mean it should be moved from required to optional (as part of `params`) which it's not part of right now.